fastjsonschema/__init__.py

```diff
@@ -98,13 +99,13 @@ def validate(definition, data, handlers={}, formats={}):
         validate({'type': 'string'}, 'hello')
         # same as: compile({'type': 'string'})('hello')
 
-    Preffered is to use :any:`compile` function.
+    Preferred is to use :any:`compile` function.
     """
-    return compile(definition, handlers, formats)(data)
+    return compile(definition, handlers, formats, **kwargs)(data)
 
 
-# pylint: disable=redefined-builtin,dangerous-default-value,exec-used
-def compile(definition, handlers={}, formats={}):
+# pylint: disable=redefined-builtin,exec-used
+def compile(definition, handlers=None, formats=None, **kwargs):
```

Comment:
Since I started refactoring your library for my own purpose and I this is my first contribution ever to open-source, I stack to the coding standards/linting we use. Since we all agree that there's only python3 left - we can leave the docs to annotations. But if you prefer mutable defaults, it is a choice

Reply:
True, Python 2 support is dropped so we could use annotations instead.